packages/sst/src/constructs/SsrSite.ts

```diff
@@ -1217,96 +1233,52 @@ function handler(event) {
         zip: { mime: "application/zip", isText: false },
       };
 
-      copy.forEach((files) => {
-        if (!files.cached) return;
+      copy.forEach(({ cached, to, versionedSubDir }) => {
+        if (!cached) return;
+
+        // Create a default file option for: unversioned files
```

Comment:
Both the `versionedSubDir` and `nonVersionedSubDir` patterns should be added to the end of the `fileOptions`.